src/Component.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\S3Writer;
+
+use Keboola\Component\BaseComponent;
+use Keboola\Component\Logger;
+
+class Component extends BaseComponent
+{
+    public function run(): void
+    {
+        /** @var Config $config */
+        $config = $this->getConfig();
+        /** @var Logger $logger */
+        $logger = $this->getLogger();
+        $writer = new S3Writer($config, $logger);
+        $writer->execute(getenv('KBC_DATADIR') . '/out/files');
```

Comment:
$this->getDatadir()

Reply:
Plus nemělo by to být /in/files ? (je to writer, tak dostává soubory na in, ne?)